apps/api/src/providers/provider.regression.ts

```diff
@@ -238,6 +240,82 @@ describe('Regression test - Providers', () => {
             expect(htmlMessage?.includes(EMAIL_BLOCK_TEXT)).to.eql(true);
             expect(messageAttachments).to.deep.equal([]);
           });
+
+          it('should send the notification to the chosen subscriber with the cc and bcc', async () => {
+            const overrides = {
+              email: {
+                cc: [MAILTRAP_EMAIL_CC],
+                bcc: [MAILTRAP_EMAIL_BCC],
+              },
+            };
+            const notification = await triggerEvent(session, providerId, subscriber.subscriberId, overrides);
+
+            expect(notification.acknowledged).to.eql(true);
+            expect(notification.status).to.eql('processed');
+            expect(notification.transactionId).to.be.a('string');
+
+            await session.awaitRunningJobs(template._id, false, 0);
+
+            const { accountId, inboxId } = getMailtrapSecrets();
+            const expectedEmails = 3;
+            const inboxMessages = await mailtrapService.pollInbox(accountId, inboxId, expectedEmails);
+            expect(inboxMessages.length).to.eql(expectedEmails);
+
+            let mainMessage;
+            let ccMessage;
+            let bccMessage;
+            for (const inboxMessage of inboxMessages) {
+              const {
+                smtp_information: {
+                  data: { rcpt_to_addrs: rcptToAddrs },
+                },
+              } = inboxMessage;
```

Comment:
```suggestion
              const recipientToAddress = inboxMessage?.smtp_information?.data?.rcpt_to_addrs;
```

A very personal thing, but I find this one more readable here :)

Reply:
I can live with it. 🙂